neard/src/cli.rs

```diff
@@ -455,10 +458,28 @@ impl RunCmd {
             }
         }
 
-        let (tx, rx) = oneshot::channel::<()>();
+        let (tx_crash, mut rx_crash) = broadcast::channel::<()>(16);
+        let (tx_dyn_configs, rx_dyn_configs) = broadcast::channel::<DynConfigs>(16);
+        let storage =
+            nearcore::open_storage(home_dir, &mut near_config).expect("storage can not access");
+        let store = storage.get_store(Temperature::Hot);
+        let dyn_configs = nearcore::dyn_config::read_dyn_configs(home_dir)
+            .unwrap_or_else(|e| panic!("Error reading dynamic configs: {:#}", e));
+        let dyn_configs_store = DynConfigStore::new(dyn_configs, consensus.clone(), tx_dyn_configs);
+        let dyn_configs_store = Arc::new(Mutex::new(dyn_configs_store));
+
+        let nearcore::config::NearConfig { validator_signer, .. } =
```

Comment:
This block of code can be moved inside the `block_on` part and I think the change for `start_with_config_and_synchronization` is unnecessary too now that `near_config.client_config.consensus` doesn't need to be set here to be a `Arc<Mutex<>>`

Reply:
Done